config/locales/de.yml

```diff
@@ -30,6 +30,7 @@ de:
     hi: 'हिंदी'
     zh-CN: 中文
   navigation:
+    google: Google
```

Comment:
I'm curious why we are moving this Google key. It seem unnecessary. Can we undo these changes?